cmd/zar/README.md

```diff
@@ -0,0 +1,340 @@
+# zar overview
+
+> DISCLAIMER: ZAR IS A CURRENTLY A PROTOTYPE UNDER DEVELOPMENT AND IS
+> CHANGING QUICKLY.  PLEASE EXPERIMENT WITH IT AND GIVE US FEEDBACK BUT
+> IT'S NOT QUITE READY FOR PRODUCTION USE. THE SYNTAX/OPTIONS/OUTPUT ETC
+> ARE ALL SUBJECT TO CHANGE.
+
+This is a sketch of an early prototype of zar and related tools for
+indexing and search log archives.
+
+We'll use the test data here:
+```
+https://github.com/brimsec/zq-sample-data/tree/master/zng
+```
+You can clone the repo or copy just this directory into your current
+directory using subversion:
+```
+svn checkout https://github.com/brimsec/zq-sample-data/trunk/zng
```

Comment:
git might be more familiar to the audience :)

Reply:
Ok, I was avoiding copying the entire data repo but you are not the first to say this so I will clarify...

Reply:
oh, i hadn't even read the sentence above. my eyes were irresistibly drawn to `svn` command to the exclusion of anything else.